subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_shadow_info

```diff
@@ -50,44 +50,54 @@ config NRF_CLOUD_SEND_SERVICE_INFO_FOTA
 	    NRF_CLOUD_FOTA_TYPE_MODEM_FULL_SUPPORTED
 
 menuconfig NRF_CLOUD_SEND_SERVICE_INFO_UI
```

Comment:
`CONFIG_NRF_CLOUD_SEND_SERVICE_INFO_UI=y` is present in `nrf_cloud_multi_service/prj.conf`. Are you leaving that so the section gets NULL'd?

Reply:
yes